### DISCUSSION TITLE
Not able to play any music

### DISCUSSION THREAD

**Discussion by PavithranGovindh** [HemantKArya/BloomeeTunes#92] [Q&A] [answered]
Not able to stream or download. I don't know what went wrong ![Screenshot_2024-12-25-10-52-27-646_ls.bloomee.musicplayer.jpg](https://github.com/user-attachments/assets/cd896ed3-bab4-4e4a-ab9d-3f04981e32ba)

![Screenshot_2024-12-25-10-52-22-436_ls.bloomee.musicplayer.jpg](https://github.com/user-attachments/assets/79328844-972c-4879-b017-1e5529edf1fd)

![Screenshot_2024-12-25-10-52-08-909_ls.bloomee.musicplayer.jpg](https://github.com/user-attachments/assets/ae67e1d9-4aa8-4fec-8532-b57f1fe36749)

Using latest version and tried both universal and v8a apk

**Comment by HemantKArya** [comment]
Hi, @PavithranGovindh thank you so much for informing the bug.
Can you let me know by trying out the latest version and inform here. https://github.com/HemantKArya/BloomeeTunes/releases/tag/v2.10.11%2B155

**Reply by PavithranGovindh** [reply]
Brother ,the app is working fine for me now, 
Checked both universal version and v8a apk

**Reply by HemantKArya** [reply, marked as answer]
Thanx for informing @PavithranGovindh .
